root/edit/details/ReorderArt.js

```diff
@@ -0,0 +1,94 @@
+/*
+ * @flow strict
+ * Copyright (C) 2020 MetaBrainz Foundation
+ *
+ * This file is part of MusicBrainz, the open internet music database,
+ * and is licensed under the GPL version 2, or (at your option) any
+ * later version: http://www.gnu.org/licenses/gpl-2.0.txt
+ */
+
+import {Artwork} from '../../components/Artwork.js';
+import DescriptiveLink
+  from '../../static/scripts/common/components/DescriptiveLink.js';
+import expand2html from '../../static/scripts/common/i18n/expand2html.js';
+import entityHref from '../../static/scripts/common/utility/entityHref.js';
+
+type Props = {
+  +archiveName: 'cover' | 'event',
+  +edit:
+    | ReorderCoverArtEditT
+    | ReorderEventArtEditT,
+  +entityType: 'event' | 'release',
+  +formattedEntityType: string,
+};
+
+const ReorderArt = ({
+  archiveName,
+  edit,
+  entityType,
+  formattedEntityType,
+}: Props): React$Element<'table'> => {
+  const display = edit.display_data;
+  const oldArt = display.old;
+  const newArt = display.new;
+  let historyMessage;
+
+  if (entityType === 'event') {
+    historyMessage = expand2html(
+      l(`We are unable to display history for this event
+         art. For a current listing of evebt art, please see the
```

Comment:
evebt :)

More generally, I wonder if we could share the string somehow.

Reply:
(Ignore my previous question then. :))

What about something like this?
```
We are unable to display history for this piece of artwork.
For what's available, see the {artpage|current listing of artwork}.
```

Reply:
Maybe? :) If @yvanzo and @Aerozol are also ok with it. I think specifying "the release" / "the event" doesn't help a lot here, anyway, but I guess just changing it with "the entity" might be too technical?

Reply:
I'm assuming the context this is displayed in shows the event/release name somewhere on the page? In which case I'm fine with leaving it out.

Otherwise some tweaks could make it work:
`We are unable to display the history for this artwork*.
See the {artpage|current artwork} for this {entitytype}.`

p.s. If I remember correctly we were trying to avoid using 'entity' in user-facing situations?
p.p.s. perhaps 'file' instead of artwork, but that might be opening a can of worms elsewhere

Reply:
That tweak would probably translate very badly in languages like Estonian that require the entity type name to be inflexed / suffixed in some way.

Reply:
True! In any case, I think it can probably be left simple. I can't think of a scenario where someone is lost and desperately needs to know if it's a event or event image without clicking the link or checking the URL or header or something (as I type this I slowly become more sure that this statement will bite me in the ass, but whatever)

Maybe it could still be shortened, but I don't think it's a big deal if not

 ```
We are unable to display the history for this artwork.
See all {artpage|current artwork}.
 ```

Reply:
I went with:
```
      l(`We are unable to display history for this piece of artwork.
         See {artpage|all current artwork}.`),
```
So that the first sentence matches the existing string, `l('We are unable to display history for this piece of artwork.')` below. We could change both if you much prefer the shorter sentence, though.